src/kudu/util/debug-util.cc

```diff
@@ -42,7 +42,9 @@
 #include <glog/logging.h>
 #include <glog/raw_logging.h>
 #ifdef __linux__
+#ifndef UNW_LOCAL_ONLY
 #define UNW_LOCAL_ONLY
+#endif
```

Comment:
fbcode seems to define this on the cmd line so we avoid redefinition is already defined.